### PR TITLE
update apis for view evaluation template screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create View Evaluation Template Screen #457](https://github.com/nerubia/kh360-react/issues/457)

FE PR: [add view evaluation template screen #476](https://github.com/nerubia/kh360-react/pull/476)

### CODE REVIEW

File: src/controllers/user/evaluation-template-contents-controller.ts [hunk 1]
```diff
@@ -27,3 +27,52 @@ export const index = async (req: Request, res: Response) => {
     res.status(500).json({ message: "Something went wrong" })
   }
 }
+
+/**
+ * Update an existing evaluation template by ID.
```
Comment (jatnerubia) [inline src/controllers/user/evaluation-template-contents-controller.ts:32]: shouldn't this be `evaluation template content` ?

File: src/controllers/user/evaluation-template-contents-controller.ts [hunk 2]
```diff
@@ -27,3 +27,52 @@ export const index = async (req: Request, res: Response) => {
...
+ * @param req.params.id - The unique ID of the evaluation template.
```
Comment (jatnerubia) [inline src/controllers/user/evaluation-template-contents-controller.ts:33]: same

File: src/controllers/user/evaluation-template-contents-controller.ts
Comment (jatnerubia) [inline src/controllers/user/evaluation-template-contents-controller.ts:1]: could you remove the "s" for now

File: src/routes/user/evaluation-template-contents-route.ts
Comment (jatnerubia) [inline src/routes/user/evaluation-template-contents-route.ts:1]: could you remove the "s" for now

File: src/routes/user/evaluation-template-contents-route.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import express from "express"
+import * as EvaluationTemplateContentsController from "../../controllers/user/evaluation-template-contents-controller"
```
Comment (jatnerubia) [inline src/routes/user/evaluation-template-contents-route.ts:2]: could you remove the "s" for now

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**